profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestInvoiceVoid.java

```diff
@@ -146,6 +146,88 @@ public void testInvoiceVoidWithPartialPay() throws Exception {
 
     }
 
+
+    @Test(groups = "slow", description = "Void a child invoice")
+    public void testChildVoidInvoice() throws Exception {
```

Comment:
Might be better to move this test under `TestIntegrationParentInvoice` since this is an HA test.